src/index.html

```diff
@@ -20,8 +20,373 @@
       href="styles/index.scss"
     />
```

Comment:
The link to the stylesheet should have an extension of `.css` instead of `.scss`. The browser can only understand CSS files directly, while SCSS is a preprocessor language that needs to be compiled into CSS.